infrastructure/local/.env

```diff
@@ -1,3 +1,3 @@
-PRISM_AGENT_VERSION=1.16.0
+PRISM_AGENT_VERSION=1.16.4-SNAPSHOT
```

Comment:
Is this a mistake in the local .env file?

Reply:
It's the latest local version. Do you want me to change it back?

Reply:
I moved it back. This file will be updated automatically in the future with each release to always correspond to the most recent version of the agent.